types/src/schemas/stoatConfigSchemaRendered.json

```diff
@@ -221,6 +309,9 @@
           "items": {
             "$ref": "#/definitions/job_runtime_entry"
           }
+        },
+        "chart_url": {
```

Comment:
naming is diverging a bit with `link` and `url`

Reply:
Good catch. I meant to change it to `chart_link` but forgot.